pkg/controllers/provisioning/provisioner.go

```diff
@@ -214,23 +214,31 @@ func (p *Provisioner) GetPendingPods(ctx context.Context) ([]*v1.Pod, error) {
 			continue
 		}
 
-		// We have pending pods that have preferred anti-affinity or topology spread constraints.  These can interact
-		// unexpectedly with consolidation so we warn once per hour when we see these pods.
-		if po.Spec.Affinity != nil && po.Spec.Affinity.PodAntiAffinity != nil {
-			if len(po.Spec.Affinity.PodAntiAffinity.PreferredDuringSchedulingIgnoredDuringExecution) != 0 {
-				if p.cm.HasChanged(string(po.UID), "pod-antiaffinity") {
-					logging.FromContext(ctx).Infof("pod %s has a preferred Anti-Affinity which can prevent consolidation", client.ObjectKeyFromObject(&po))
-				}
+		p.consolidationWarnings(ctx, po)
+		pods = append(pods, &po)
+	}
+	return pods, nil
+}
+
+// consolidationWarnings potentially writes logs warning about possible unexpected interactions between scheduling
+// constraints and consolidation
+func (p *Provisioner) consolidationWarnings(ctx context.Context, po v1.Pod) {
+	// We have pending pods that have preferred anti-affinity or topology spread constraints.  These can interact
+	// unexpectedly with consolidation so we warn once per hour when we see these pods.
+	if po.Spec.Affinity != nil && po.Spec.Affinity.PodAntiAffinity != nil {
+		if len(po.Spec.Affinity.PodAntiAffinity.PreferredDuringSchedulingIgnoredDuringExecution) != 0 {
+			if p.cm.HasChanged(string(po.UID), "pod-antiaffinity") {
+				logging.FromContext(ctx).Infof("pod %s has a preferred Anti-Affinity which can prevent consolidation", client.ObjectKeyFromObject(&po))
```

Comment:
```suggestion
				logging.FromContext(ctx).Infof("pod %s has a preferred antiAffinity which can prevent consolidation", client.ObjectKeyFromObject(&po))
```

nit: Prefer lowercase logging?

Reply:
Disagree w/ the recommendations -- I think @tzneal got it right here.. Proper nouns make sense to be capitalized. Starting each log line with a capital (as if a sentence), make less sense.

Reply:
I suppose the camelCased version is the same as customers would write into their yaml.

Reply:
> I'm not really sure what the proper "logging standards"

![image](https://user-images.githubusercontent.com/2585992/208524167-1904f140-aadf-4605-8cb0-093032777361.png)